controls/automake_works.rb

```diff
@@ -0,0 +1,28 @@
+title 'Tests to confirm automake works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'automake')
+
+control 'core-plans-automake-works' do
+  impact 1.0
+  title 'Ensure automake works as expected'
+  desc '
+  Verify automake by ensuring (1) its installation directory exists and (2) that
+  it returns the expected version
+  '
+  
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+  
+  plan_pkg_ident = ((plan_installation_directory.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
+  plan_pkg_version = (plan_pkg_ident.match /^#{plan_origin}\/#{plan_name}\/(?<version>.*)\//)[:version]
+  describe command("DEBUG=true; hab pkg exec #{plan_pkg_ident} automake --version") do
```

Comment:
Do we need DEBUG=true for this command? 

Here's the output I get without it being set:
`automake (GNU automake) 1.16.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.`

Looks like it covers this control.

Reply:
We should also avoid using hab pkg exec for testing binaries since we want to confirm they are binlinked correctly. `/bin/automate --version` should suffice.

Reply:
Hi Steven.  All addressed in latest commit.

Reply:
Done, thanks.